src/wallet/wallet.cpp

```diff
@@ -2173,6 +2172,7 @@ bool CWallet::CommitTransaction(CWalletTx& wtxNew, CReserveKey& reservekey)
             if (!wtxNew.AcceptToMemoryPool(false))
             {
                 // This must not fail. The transaction has already been signed and recorded.
+                // Also the fee never exceeds maxTxFee which forces mempool rejection
```

Comment:
But, please, don't use acronyms for the name functions in the comments: the full name of the function is something that can be seen in grep.